packages/swiper/src/carousel.component.ts

```diff
@@ -161,6 +196,13 @@ Stratus.Components.SwiperCarousel = {
                             preppedImage.src = image
                         } else if (typeof image === 'object') {
                             if (Object.prototype.hasOwnProperty.call(image, 'src')) {
+                                if (
+                                    Object.prototype.hasOwnProperty.call(image, 'lazy')
```

Comment:
A little code golf:

- You can replace the `Object.prototype.hasOwnProperty.call` with `_.has`, which is shorter and doesn't the same thing.
- You can also combine both the `_.has` and the comparison into a getter like so: `_.get(image, 'lazy') === 'stratus-src`, which can also use any normal dot or bracket notation for deep values without any catastrophic failures.